### ISSUE TITLE
Search for add-ons in the add-ons manager, and show the number of pieces

### ISSUE THREAD

**Issue by NVins** [aminomancer/uc.css.js#90]
It would be nice,  ifcould search for add-ons in the add-ons manager, and also show the number of add-ons.
Thank you!
![1](https://github.com/aminomancer/uc.css.js/assets/157918101/0959eceb-7e5c-4e2a-8cf5-eaef13eb0f41)

**Comment by aminomancer** [comment]
Is this filed on bugzilla also? Seems more like a firefox enhancement than a third-party mod.

The number should maybe be moved into the searchbar placeholder. It looks awkward being so prominent since most people don't need to know how many add-ons they have. The placeholder could say "Search 24 add-ons"

**Comment by NVins** [comment]
> Is this filed on bugzilla also? Seems more like a firefox enhancement than a third-party mod.
> 
> The number should maybe be moved into the searchbar placeholder. It looks awkward being so prominent since most people don't need to know how many add-ons they have. The placeholder could say "Search 24 add-ons"

In Bazilla, no, of course, it's good to have the search for add-ons in the extension manager in Firefox itself, but hopefully at least in the extension options panel.
The display of the number of add-ons can be made with the possibility of disabling it, or made less visible, for example, at the bottom or, as you suggest, in the search.

**Comment by aminomancer** [comment]
Idk about this. If the panel needs to have every feature of about:addons, it might as well just be about:addons. Point of this script is to be a lightweight alternative. Searching is really the least lightweight thing possible. That requires a searching algorithm, and what even happens when you search? You type something, hit enter, then what? Everything that doesn't match is hidden?

I have very limited time so I prefer to focus on things that will have high impact. Searchbar or number display in this script (which is already deprecated since Firefox added the unified extensions panel) seems pretty low impact. I'm not sure what purpose number display even serves. It doesn't hurt to have it, but that kind of "why not?" mentality is more appropriate for a page with unlimited space, like about:addons. Popup panels, conversely, call for a "why?" mentality. Especially when they're maintained by 1 person with very limited time 😅

**Comment by NVins** [comment]
> Idk about this. If the panel needs to have every feature of about:addons, it might as well just be about:addons. Point of this script is to be a lightweight alternative. Searching is really the least lightweight thing possible. That requires a searching algorithm, and what even happens when you search? You type something, hit enter, then what? Everything that doesn't match is hidden?
> 
> I have very limited time so I prefer to focus on things that will have high impact. Searchbar or number display in this script (which is already deprecated since Firefox added the unified extensions panel) seems pretty low impact. I'm not sure what purpose number display even serves. It doesn't hurt to have it, but that kind of "why not?" mentality is more appropriate for a page with unlimited space, like about:addons. Popup panels, conversely, call for a "why?" mentality. Especially when they're maintained by 1 person with very limited time 😅

Yes, when you open the Extension Manager, you just need to enter the name right away, or click on the search button, and it will find the add-on by the name you entered. You have understood everything correctly.
